src/bootstrap.php

```diff
@@ -1556,11 +1557,20 @@ function (Request $request, string $id) use ($app) {
 
         $version = (int) $type->getParameter('version');
         $type = $type->getType();
+        $headers = ['Content-Type' => sprintf('%s; version=%s', $type, $version)];
+
+        if ($version < 2) {
+            if (empty($packages['relatedContent'])) {
+                throw new UnsupportedVersion('Deprecation: `relatedContent` is no longer required.');
+            } else {
+                $headers['Warning'] = '299 elifesciences.org "Deprecation: `relatedContent` is no longer required."';
```

Comment:
The message should be something like 'Deprecation: Support for version 1 will be removed'.